packages/utils/src/schema/getDefaultFormState.ts

```diff
@@ -196,7 +243,8 @@ export function computeDefaults<
   switch (getSchemaType<S>(schema)) {
     // We need to recur for object schema inner default values.
     case "object":
-      return Object.keys(schema.properties || {}).reduce(
+      // eslint-disable-next-line no-case-declarations
```

Comment:
Wrap this case in pair of brackets to make this lint error go away: https://eslint.org/docs/latest/rules/no-case-declarations

Reply:
done